internal/cmd/commands/database/migrate.go

```diff
@@ -0,0 +1,253 @@
+package database
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/boundary/internal/cmd/base"
+	"github.com/hashicorp/boundary/internal/cmd/config"
+	"github.com/hashicorp/boundary/internal/db/schema"
+	"github.com/hashicorp/boundary/sdk/wrapper"
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/hashicorp/vault/sdk/helper/mlock"
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+var (
+	_ cli.Command             = (*MigrateCommand)(nil)
+	_ cli.CommandAutocomplete = (*MigrateCommand)(nil)
+)
+
+type MigrateCommand struct {
+	*base.Command
+	srv *base.Server
+
+	SighupCh   chan struct{}
+	ReloadedCh chan struct{}
+	SigUSR2Ch  chan struct{}
+
+	Config *config.Config
+
+	configWrapper wrapping.Wrapper
+
+	flagConfig             string
+	flagConfigKms          string
+	flagLogLevel           string
+	flagLogFormat          string
+	flagMigrationUrl       string
+	flagAllowDevMigrations bool
+}
+
+func (c *MigrateCommand) Synopsis() string {
+	return "Initialize Boundary's database"
```

Comment:
This is the same synopsis as the `database init` command. Should it be different?

Reply:
Changed it. Thanks!